src/models/train_model.py

```diff
@@ -54,43 +48,29 @@ def forward(self, x):
         return F.softmax(x, dim=1)
 
 class Model():
-    def __init__(self):
-        """Constuctor method for TrainModel class can tweak epoch and batch_size"""
+    def __init__(self, batch_size, epochs, learning_rate, validation_percent, training_data):
```

Comment:
Nice call adding these, I think this will help us run different versions of the model.